app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from typing import Dict, Any
+from app.errors import (NotVaccinatedError,

```

Comment:
According to the checklist, you should use parentheses when importing multiple modules. It is better to use parentheses for better readability.